pkg/controller/rest/verifiable/operation_test.go

```diff
@@ -296,6 +296,52 @@ func TestGetCredentials(t *testing.T) {
 	})
 }
 
+func TestGeneratePresentation(t *testing.T) {
+	t.Run("test generate presentation - success", func(t *testing.T) {
+		cmd, err := New(&mockprovider.Provider{
```

Comment:
you can reuse setup - same comment as above

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/f5ef9747bc23d5292e75bd2afabead5d2314681f..d8e1c2fd1338d615a9aeb8420119c409e7097a44#diff-4fe89d968296211ff49175a4a8dba6c9R300).